.github/workflows/tests.yml

```diff
@@ -0,0 +1,19 @@
+name: CI Tests
+on: [push]
+
+jobs:
+  frontend:
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
As a general rule, shouldn't we pin all versions of our dependencies/os so that we have reproducible results?

Reply:
Good point. Both are pointing to ubuntu 18.04, but I went ahead to specified it to 18.04.